playbooks/aws/openshift-cluster/install.yml

```diff
@@ -21,5 +21,45 @@
 - name: run the std_include
   include: ../../common/openshift-cluster/std_include.yml
 
-- name: run the config
-  include: ../../common/openshift-cluster/config.yml
+- name: Verify Requirements
```

Comment:
If we're going to use this in multiple places, we should probably make it into it's on playbook and include here and in common/../config.yml

Reply:
The health checker play has already been converted to it's own play (`openshift-checks/install.yml`), but that change is not captured here.